packages/node-experimental/test/sdk/api.test.ts

```diff
@@ -64,7 +65,11 @@ describe('withActiveSpan()', () => {
   });
 
   it('when `null` is passed, should start a new trace for new spans', async () => {
-    const beforeSendTransaction = jest.fn(() => null);
+    const transactions: Event[] = [];
+    const beforeSendTransaction = jest.fn((event: Event) => {
+      transactions.push(event);
```

Comment:
l: why do we have `transactions` array here?

Reply:
makes it a bit easier to test what is sent than to use `hasBeenCalledWith`, as you don't have to use `objectContains()` everywhere which gives weird diffs in jest for large objects 😬